### PR TITLE
Add quorum to proposal

### PR DESCRIPTION
**Pull request by jowlee**
## Summary
* Update libs to get quorum function from gov. 
* Add binding to audius service
* Update Proposal type to have quorum amount
* Display in Proposal UI

Note: the added quorum met/not met and voted/quorum ui are only displayed if the proposal is active
Screen Shot 2021-02-25 at 11 54 39 AM
Screen Shot 2021-02-25 at 11 58 03 AM

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -151,6 +151,7 @@ export type Proposal = {
   name?: string
   description?: string
   evaluatedBlock?: Block
+  quorum: BigNumber
```
Comment (raymondjacobson) [inline src/types.ts:154]: shouldn't this be BN not BigNumber?

**Review by raymondjacobson — APPROVED** [review]
Might be worth showing the quorum even for inactive proposals

File: src/types.ts (hunk 1 above)
Comment (jowlee) [reply src/types.ts:154]: At the top we type alias BN to BigNumber

Comment (raymondjacobson) [reply src/types.ts:154]: we should probably remove at some point, but sgtm!

**Pull request merged**